index.js

```diff
@@ -164,7 +164,7 @@ module.exports = function (args, opts) {
 			next = args[i + 1];
 			if (
 				next !== undefined
-				&& !(/^-/).test(next)
+				&& !(/^(-|--)[^-]/).test(next)
```

Comment:
i notice here you're careful to disallow three or more dashes - can we add a test for that?

Reply:
Ok, I'll check how much I believe triple dash is a supported behaviour, and either add tests or explain why I think it is suspect. 😄 

(The test came from existing code.)

Reply:
Added test, as much to show short and long option now consistent as for the outcome.